### PR TITLE
plugins/blink-cmp-copilot: init

### PR DESCRIPTION
**Pull request by HeitorAugustoLN**

### CODE REVIEW

File: plugins/by-name/blink-cmp-copilot/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          completions properly appearing in blink-cmp-copilot.
+        ''
+      ]
+      ++ lib.optionals (isEnabled copilot-lua-cfg.panel.enabled) [
+        ''
+          It is recommended to disable copilot's `panel` module, as it can interfere with completions
+          properly appearing in blink-cmp-copilot.
+        ''
+      ];
+    plugins.copilot-lua.enable = lib.mkDefault true;
```
Comment (traxys) [inline plugins/by-name/blink-cmp-copilot/default.nix:30]: Why put it as `lib.mkDefault true` and not simply `true`? I'd think a user would be warned if they explicitely disable the plugin, I would prefer requiring `lib.mkForce`

File: plugins/by-name/blink-cmp-copilot/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "blink-cmp-copilot";
+  package = "blink-cmp-copilot";
+
+  maintainers = [ lib.maintainers.HeitorAugustoLN ];
+
+  callSetup = false;
+  hasLuaConfig = false;
+  hasSettings = false;
+
```
Comment (traxys) [inline plugins/by-name/blink-cmp-copilot/default.nix:11]: Could you add `description = '...'` that says that this plugin is configured through blink.cmp's `sources.providers` ?

File: plugins/by-name/blink-cmp-copilot/default.nix (hunk 1 above)
Comment (HeitorAugustoLN) [reply plugins/by-name/blink-cmp-copilot/default.nix:30]: To be the same behavior as current copilot-cmp: https://github.com/nix-community/nixvim/blob/8938e09db14d510dcc2f266e8b2e738ee527d386/plugins/by-name/copilot-cmp/default.nix#L72

File: plugins/by-name/blink-cmp-copilot/default.nix (hunk 2 above)
Comment (HeitorAugustoLN) [reply plugins/by-name/blink-cmp-copilot/default.nix:11]: Done

**Review by traxys — APPROVED** [review]

**Comment by traxys** [comment]
@mergify queue

**Pull request merged**